### DISCUSSION TITLE
API Endpoints for Past Precipitation Data

### DISCUSSION THREAD

**Discussion by behrad710** [weather-gov/api#583] [Q&A]
I'm seeing a discrepancy between the data I generate manually from column 7 [here](https://www.dropbox.com/s/csxlv7rmpwyaxzh/NOAA%20Past%20Weather%203.JPG?dl=0) and the data I pull from [this api](https://streaklinks.com/BNeujyyAgFCoksjc4Qd74BNC/https%3A%2F%2Fwww.ncdc.noaa.gov%2Fcdo-web%2Fwebservices%2Fv2%23gettingStarted). I assume I'm using the wrong endpoint?

My hope is to be able to tap into whatever API will give me [this data](https://www.dropbox.com/s/csxlv7rmpwyaxzh/NOAA%20Past%20Weather%203.JPG?dl=0). Where are the specific docs for that? Can you point me in the right direction?

Thanks!

**Comment by gutzbenj** [comment]
Dear @behrad710 , what resolution of precipitation data do you need? NOAAs GHCN provides global daily historical data at https://www.ncei.noaa.gov/products/land-based-station/global-historical-climatology-network-daily .
